critiquebrainz/frontend/templates/macros.html

```diff
@@ -35,3 +35,27 @@
     <abbr title="{{ _('Submit and delete your votes.') }}">{{ _('Review votes') }}</abbr>
   {% endif %}
 {% endmacro %}
+
+{% macro coverart(entity_id, entity_type, attributes='') %}
+  {# Note: when passing in `attributes`, don't quote attribute values; they will be added automatically #}
+  {% if entity_type == 'event' %}
+    <img src="/static/img/event-placeholder.png" {{ attributes }} />
+  {% else %}
+    <img {{ attributes }} />
+    <script>
+      $("img").last()
+          .on("error", function () {
+            $(this).attr("src", "/static/img/missing-art.png");
+          })
+          .attr("src", "//coverartarchive.org/release-group/{{ entity_id }}/front-250");
+    </script>
+  {% endif %}
+{% endmacro %}
+
+{% macro review_byline(review) %}
+{{ _(
+  '%(entitytype)s review by %(user)s',
```

Comment:
That may not be ideal for translators. The strings `_('Event')` and `_('Music')` below have no context, and may be translated differently alone than they would be in this sentence. I'd use separate strings for each entity type.